### PR TITLE
Focus title input when clicking helper link

### PR DESCRIPTION
**Pull request by JulienTant**
Improve UX as suggested by @lafriks by setting the focus back on the title input after clicking the `Start the title with WIP:` helper link.

### CODE REVIEW

File: public/js/index.js [hunk 1]
```diff
@@ -1660,6 +1659,7 @@ function initWipTitle() {
         e.preventDefault();
 
         var $issueTitle = $("#issue_title");
+        $issueTitle.focus();
```
Comment (lafriks) [inline public/js/index.js:1662]: It would be better to focus field after value has been set

Comment (JulienTant) [reply public/js/index.js:1662]: If I do that, then the cursor will be at the beginning of the input. By focusing before changing the content of the field, I make sure that my cursor will be at the end of it, which is preferable i think.

Comment (lafriks) [reply public/js/index.js:1662]: Yes it should be at the end. For me it worked also if focused after value is set but ok, NVM than :)

**Review by lafriks — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Pull request merged**